src/documents/CoreAPI.md

```diff
@@ -0,0 +1,635 @@
+---
+title: Core API
+layout: default
+category: Infusion
+---
+
+# Infusion Core API
```

Comment:
The Markdown files shouldn't have a level one header explicitly. The text in the "title" attribute above will be used as the level one header for the page in the rendered HTML.